scripts/shelley-testnet2.sh

```diff
@@ -19,19 +19,45 @@ genesis_hash="$(cabal new-exec -- cardano-cli --real-pbft print-genesis-hash --g
 
 ALGO="--real-pbft"
 NOW=`date "+%Y-%m-%d 00:00:00"`
-NETARGS=(
+NETARGS1=(
```

Comment:
@Jimbo4350, can we factor the common part & concatenate it with the desired variable part?  because it's sligtly unwieldy now..

Reply:
Updated!